tests/cross/src/test/scala/tests/pc/CompletionKeywordSuite.scala

```diff
@@ -427,7 +383,7 @@ class CompletionKeywordSuite extends BaseCompletionSuite {
     // to avoid newMain annotation
     filter = str => !str.contains("newMain"),
   )
-
+  // couldnt' reproduce
```

Comment:
```suggestion
```
We shouldn't show anything here since `new` is not a start for any of the completions. You can do a TODO issue to fix it later

Reply:
I will add a TODO, because it's a weird case, since `new` is not shown in `path` and the tree looks the same as for `case @@`

Reply:
Might need the CURSOR trick